test/unit/data/attachments.js

```diff
@@ -16,10 +16,7 @@ describe('.zip attachments streaming', () => {
       if(err) return done(err);
 
       result.filenames.should.eql([
-        // eslint-disable-next-line no-trailing-spaces
-        'media/firstfile.ext', 
```

Comment:
This line has been lost, it needs re-instating.  `make test-unit` fails.  But **_why has this not broken CI?_**